adaptive/learner/learner1D.py

```diff
@@ -590,12 +590,10 @@ def plot(self):
             Plot of the evaluated data.
         """
         hv = ensure_holoviews()
-        if not self.data:
-            p = hv.Scatter([]) * hv.Path([])
-        elif not self.vdim > 1:
+        if not self.vdim > 1:
```

Comment:
`if self.vdim <= 1`

Reply:
Ehm, `== 1`, I guess we don't allow `vdim == 0`?

Reply:
I vaguely remember that `vdim` could be 0, but looking at the code I don't see how (and why).

I am a bit hesitant to remove it.

Reply:
Even then `<=` is easier to read than `not >`.

Reply:
Right, I amended the commit.